spec/swagger.json

```diff
@@ -0,0 +1,110 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Sic-Code-API",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "http://api.chs.local/siccode-api",
+      "description": "Development server"
+    }
+  ],
+  "paths": {
+    "/search": {
+      "post": {
+        "summary": "Search for sic code via economic activity",
+        "operationId": "searchSicCodes",
+        "description": "Return all matching sic code records that correspond with the search terms. This will be ordered by most number of hits (for example, results with 2 keyword matches will be retuned before matches with 1 keyword match)",
```

Comment:
Generally speaking, if numbers are less than 10, write them out. Thus 2 keyword -> two keyword and 1 keyword -> one keyword.
Typo:
retuned -> returned.